src/lib.rs

```diff
@@ -186,7 +186,7 @@ pub fn parse_x509_certificate(i: &[u8]) -> X509Result<X509Certificate> {
 /// This function is an alias to [CertificateRevocationList::from_der](revocation_list::CertificateRevocationList::from_der). See this function
 /// for more information.
 #[inline]
-pub fn parse_x509_crl(i: &[u8]) -> X509Result<CertificateRevocationList> {
+pub fn parse_x509_crl<'i>(i: &'i [u8]) -> X509Result<'i, CertificateRevocationList<'i>> {
     CertificateRevocationList::from_der(i)
 }
 
```

Comment:
Does this need to be a named lifetime? It seems like it could be elided like the est.

```suggestion
pub fn parse_x509_crl(i: &[u8]) -> X509Result<'_, CertificateRevocationList<'_>> {
    CertificateRevocationList::from_der(i)
}
```

Reply:
I has the same question :) I have no real preference, I just found strange to use anonymous lifetimes while the lifetime is known to be exactly the same as input. So, no real preference here